src/ripple/app/tx/TransactionAcquire.cpp

```diff
@@ -124,6 +124,7 @@ void TransactionAcquire::trigger (Peer::ptr const& peer)
         protocol::TMGetLedger tmGL;
         tmGL.set_ledgerhash (mHash.begin (), mHash.size ());
         tmGL.set_itype (protocol::liTS_CANDIDATE);
+        tmGL.set_querydepth (3); // We probably need the whole thing
```

Comment:
Oh, here's 3.  Well, I don't get it then.  Why are we setting the query depth in two different places?